contributing-guides/style-guide.md

```diff
@@ -67,3 +67,12 @@ Keep the following guidelines in mind when choosing tokens:
 
 In general, tokens should make it as intuitive as possible
 to figure out how to use the command and fill it in with values.
+
+More technical wording on description lines should use the `backtick` syntax.
+Use backticks on the following:
+
+1. Paths, ex. `package.json`, `/etc/package.json`.
+2. Extensions, ex. `.dll`.
+3. Commands, ex. `ls`.
+4. Keyboard commands, ex. `CTRL+ALT+DEL`.
```

Comment:
I don't think this works very well. Backticks indicate a file or command, and this is neither.

Reply:
Isn't it a command?

Reply:
Actually, after some thought, I think I agree that this should not be surrounded by backticks.
Removed it from the style guide.